src/slider.jsx

```diff
@@ -319,17 +311,75 @@ const Slider = React.createClass({
       document.addEventListener('touchcancel', this._dragTouchEndHandler, false);
     }
     this._onDragStart(event);
+
+    // Cancel scroll and context menu
+    event.preventDefault();
   },
 
   _onHandleMouseDown(event) {
     if (document) {
       document.addEventListener('mousemove', this._dragHandler, false);
       document.addEventListener('mouseup', this._dragEndHandler, false);
-      this._toggleSelection('none');
+
+      // Cancel text selection
+      event.preventDefault();
+
+      // Set focus manually since we called preventDefault()
+      this.refs.handle.focus();
     }
     this._onDragStart(event);
   },
 
+  _onHandleKeyDown(event) {
+    const decrease =
+      keycode(event) === 'page down' ||
+      keycode(event) === 'home' ||
+      keycode(event) === 'left' ||
+      keycode(event) === 'down';
+
+    const increase =
+      keycode(event) === 'page up' ||
+      keycode(event) === 'end' ||
+      keycode(event) === 'right' ||
+      keycode(event) === 'up';
```

Comment:
Nice, I like it! can I just change `let action = false` to `let action = ''`? I like of having variables with the same type always